### DISCUSSION TITLE
primefaces 13 missing layout css

### DISCUSSION THREAD

**Discussion by atomz4peace** [primefaces/community#2574] [PrimeFaces] [answered]
I tried a very basic setup using prime 13, java 8, wildfly 20, mojarra 2.3. This is my xhtml file:

```
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:ui="http://java.sun.com/jsf/facelets"
	xmlns:c="http://java.sun.com/jsp/jstl/core"
	xmlns:h="http://java.sun.com/jsf/html"
    xmlns:p="http://primefaces.org/ui">
    <h:head>
    </h:head>
    <h:body>
        <p:spinner />
        <p:commandButton value="Button"/>
	<h:outputText value="Testing" />
    </h:body>
</html>
```

The docs say that's all that is needed. And sure enough, the spinner and button paint and have style. But the plain body does not. It shows the browser default of times. I see in the showcase that my browser loads several other layout css files. But my simple test loads only the theme and components css but those do not define any style for the plain body like font or anything. 

I wanted to start with the default look and feel of the showcase. I'm I just missing something simple to "turn on" the full styling? 

Thank you!

**Comment by melloware** [comment, marked as answer]
You have it correct the themes built into PF like Arya, Saga, Vela etc only style the components they **DO NOT** style your Body or Page etc.   The premium templates like Poseidon, Freya , Diamond do that with `layout.css` etc.  

For example here is Diamond: https://www.primefaces.org/diamond/

If not just take the layout.css from the Showcase its right here.

https://www.primefaces.org/showcase/javax.faces.resource/css/layout/layout.css.xhtml?ln=showcase

Just download that raw file and save as a CSS file and load it in your app.

**Comment by atomz4peace** [comment]
Got it, thank you!
